src/main/java/com/revature/services/RSSService.java

```diff
@@ -34,6 +35,18 @@ public RSSService(RestTemplateBuilder restTemplateBuilder) {
 		this.restTemplate = restTemplateBuilder.build();
 	}
 	
+	@Value("${environments.rss.login}")
```

Comment:
We really only need one environment variable that gives us the location of the server. We can embed the API strings from there directly. So we should get something like `@Value("${environments.rss-service-url}") String rssServiceUrl`, then later use it as `String loginUri = rssServiceUrl + "/user/login";`